packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -159,9 +186,28 @@ const ComboBoxButton = React.forwardRef(function ComboBoxButton(props: ComboBoxB
       valueId,
       validationState === 'invalid' ? invalidId : null
     ].filter(Boolean).join(' '),
-    elementType: 'div'
+    elementType: 'div',
+    // Prevent focus on press and focus on press end to avoid situations where user
+    // presses and holds on the combobox button and tray immediately opens due to menuTrigger=focus
+    // @ts-ignore
```

Comment:
i'm surprised you need this?

Reply:
yeah, I was too but it turns out we don't document `preventFocusOnPress` in the types cuz we probably want it to be internal:
https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/button/src/useButton.ts#L57-L58

Reply:
Should we even support `menuTrigger = focus` for mobile combo box? That seems weird in general. Generally buttons don't do anything on focus.

Reply:
I'm fine with removing this option for mobile comboboxes, I'll make a note to make a follow up PR to mention this in the docs.